astropy/modeling/fitting.py

```diff
@@ -969,14 +969,18 @@ def _wrap_deriv(params, model, weights, x, y, z=None):
 
 class SLSQPLSQFitter(Fitter):
     """
-    SLSQP optimization algorithm and least squares statistic.
-
+    Sequential Least Squares Programming (SLSQP) optimization algorithm and
+    least squares statistic.
 
     Raises
     ------
     ModelLinearityError
         A linear model is passed to a nonlinear fitter
 
+    Notes
+    ------
+    See https://docs.astropy.org/en/stable/api/astropy.modeling.optimizers.SLSQP.html
```

Comment:
I think @astrofrog meant some external literature to explain the algorithm, not a link back to this package's documentation.

Reply:
Hmm, OK. I'll see what I can find 👍

Reply:
The optimizer (linked to above) has a reference to the algorithm description and linking to it is how I read [this comment](https://github.com/astropy/astropy/issues/9949#issuecomment-586369218). I think it's OK to link to the optimizer in this package to pick up any specifics in setting up and calling the scipy algorithm.

Reply:
In that case, please use the Sphinx `:ref:` linking, not hardcoding the URL. Thanks!